conf/default.cfg

```diff
@@ -130,6 +130,26 @@ mask = 0xFFFFFFFFFF0000FF
 # How to format the container name (python string format)
 format = %016d
 
+
+[filter:container_hierarchy]
+# Spawn containers for each subdirectory, instead of emulating
+# directories with '/' in object names. This middleware must be
+# placed after authentication in the pipeline.
+
+use = egg:oioswift#container_hierarchy
+#log_level = INFO
+
+# These options have the same meaning as in hashedcontainer.
+# When using swift3, all 3 must be true.
```

Comment:
Anyway to check this when pipeline is created ?

Reply:
I just added a method to do some checks on the pipeline when starting.